python/ray/serve/_private/deployment_state.py

```diff
@@ -1281,21 +1254,11 @@ def __init__(
         self._last_broadcasted_running_replica_infos: List[RunningReplicaInfo] = []
         self._last_broadcasted_deployment_config = None
 
-    @property
-    def autoscaling_policy_manager(self) -> AutoscalingPolicyManager:
-        return self._target_state.info.autoscaling_policy_manager
-
     def should_autoscale(self) -> bool:
         """
         Check if the deployment is under autoscaling
         """
-        return self.autoscaling_policy_manager.should_autoscale()
-
-    def get_autoscale_metric_lookback_period(self) -> float:
```

Comment:
Was this never used? I don't think we've added it anywhere in the new changes.

Reply:
Yeah I think it wasn't used